Add error handler warning to update guide (#2841)
ibexa/documentation-developer: docs/update_and_migration/from_4.6/update_from_4.6.md

## v4.6.8

!!! caution

>     To avoid deprecations when updating from an older PHP version to PHP 8.2 or 8.3, run the following commands:

Reviewer: ```suggestion
    To avoid deprecations when using PHP 8.2 or 8.3, run the following commands:
```
